app/uk/gov/hmrc/childcarecalculatorfrontend/models/schemes/EmploymentSupportedChildcare.scala

```diff
@@ -25,10 +25,13 @@ class EmploymentSupportedChildcare extends Scheme {
 
   override def eligibility(answers: UserAnswers): Eligibility = {
     val No = YesNoUnsureEnum.NO.toString
```

Comment:
Let's use the constants we've already defined in childcareconstants

Reply:
Done